app/controllers/admin/facilities_controller.rb

```diff
@@ -108,7 +108,9 @@ def upload
 
   def new_facility(attributes = nil)
     @facility_group.facilities.new(attributes).tap do |facility|
-      facility.country ||= Rails.application.config.country[:name]
+      facility.district ||= @facility_group.name
+      facility.state ||= @facility_group.region.state.name if Flipper.enabled?(:regions_prep)
```

Comment:
Instead of gating this on the feature flag, could you instead just gate it on the existence of the the state region?  That way the state will populate correctly as soon as regions are filled.

Reply:
We're using the `regions_prep` flag to mean: "Region backfill has been run and all regions are in place". My worry with doing say `@facility_group.region&.state&.name` is that we might forget to clean it up and that extra code will sit around forever. The flag is easier to grep and clean.